rest/revocation_test.go

```diff
@@ -1441,24 +1458,32 @@ func TestRevocationWithUserXattrs(t *testing.T) {
 	})
 
 	defer rt.Close()
-
-	collection, err := base.AsCollection(rt.Bucket().DefaultDataStore())
-	if err != nil {
-		t.Skip("Test requires Couchbase Bucket")
+	collection := rt.GetSingleTestDatabaseCollection()
+	c := collection.Name()
+	s := collection.ScopeName()
+
+	var data sgbucket.DataStore
+	dataStores, _ := rt.Bucket().ListDataStores()
+	for _, dataStoreName := range dataStores {
+		if dataStoreName.ScopeName() == s && dataStoreName.CollectionName() == c {
+			data = rt.Bucket().NamedDataStore(dataStoreName)
+		}
```

Comment:
if you need the data store, we should think about allowing this a test only function. I'm reluctant to make `DatabaseCollection.dataStore` public but that might be an option also.